quickwit/quickwit-storage/src/cache/storage_with_cache.rs

```diff
@@ -71,6 +72,17 @@ impl Storage for StorageWithCache {
         }
     }
 
+    async fn get_slice_stream(
+        &self,
+        path: &Path,
+        _range: Range<usize>,
+    ) -> StorageResult<Box<dyn AsyncRead + Send + Unpin>> {
+        unimplemented!(
```

Comment:
i still think forwarding to the underlying storage was better O_o

Reply:
did I misunderstand this https://github.com/quickwit-oss/quickwit/pull/3869#issuecomment-1740164229 ?

> Yes unimplemented!() is ok here.

Reply:
I am not sure what I was thinking. Anyway let's stick to `unimplemented!()`